gradle-plugins/preview-rpc/src/test/kotlin/org/jetbrains/compose/desktop/ui/tooling/preview/rpc/utils/utils.kt

```diff
@@ -5,14 +5,18 @@
 
 package org.jetbrains.compose.desktop.ui.tooling.preview.rpc.utils
 
+import java.io.File
+
 internal fun systemProperty(name: String): String =
     System.getProperty(name) ?: error("System property is not found: '$name'")
 
 internal val isWindows =
     systemProperty("os.name").startsWith("windows", ignoreCase = true)
 
 internal val previewTestClaspath: String
-    get() = systemProperty("org.jetbrains.compose.test.rpc.classpath")
+    get() = systemProperty("org.jetbrains.compose.tests.rpc.classpath.file").let {
```

Comment:
"test." changes to "tests."
Is it correct?

Should we mention it in Update Notice?

Reply:
No, that's a property, which exists only in our tests in one subproject